tutorials/plugins/running_code_in_the_editor.rst

```diff
@@ -530,14 +530,21 @@ EditorScripts, with a ``_run()`` method already inserted:
 This ``_run()`` method is executed when you use **File > Run** or the keyboard
 shortcut :kbd:`Ctrl + Shift + X` while the EditorScript is the currently open
 script in the script editor. This keyboard shortcut is only effective when
-currently focused on the script editor.
+currently focused on the script editor. If using an external script editor, then
+you must use one of the approaches described below to run the EditorScript.
 
-Scripts that extend EditorScript must be ``@tool`` scripts to function.
+There are two other ways to run an EditorScript:
 
-.. note::
+- If the script has a ``class_name`` set at the top, it will appear in the command palette.
+  Press :kbd:`Ctrl + Shift + P` (:kbd:`Cmd + Shift + P` on macOS) to bring up
+  the command palette, then enter the class name. The script will appear under
+  the Editor Scripts category. The class name is automatically capitalized,
```

Comment:
The automatic capitalization is a bit confusing as the capitalization of the example doesn't change, a space is inserted though, I think this can be clarified further (and make it clear what happens for a class like `add_foliage`